frontend/common/src/constants/regularExpressions.tsx

```diff
@@ -0,0 +1,2 @@
+export const keyString = /^[a-z]+[_a-z0-9]*$/;
+export const phoneNumber = /^\+[1-9]\d{1,14}$/;
```

Comment:
Clearer and less typing to just keep the variable consistent and avoid aliasing?

```suggestion
export const keyStringRegex = /^[a-z]+[_a-z0-9]*$/;
export const phoneNumberRegex = /^\+[1-9]\d{1,14}$/;
```

Reply:
Further: I've gotten in the habit of adding a link to a good testing tool, with some good/bad strings, as a way to explore the regex more intuitively :)

```suggestion
# Links to regex experimentation tool are provided for each regex pattern.
# Please feel free to "Update" and liberally add candidate test strings.
# If you make changes, they'll be most discoverable by updating the link below.

# See: https://regex101.com/r/TEDtDo/1
export const keyStringRegex = /^[a-z]+[_a-z0-9]*$/;
# See: https://regex101.com/r/EAZ8ha/1
export const phoneNumberRegex = /^\+[1-9]\d{1,14}$/;
```

Reply:
These are great suggestions!  I'm putting them in.

Reply:
Nice, this is a great idea!